### PR TITLE
SubscriptionsImportFragment: Inline setup stuff

### PR DESCRIPTION
**Pull request by Profpatsch**
Very small simplification

#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [x] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionsImportFragment.java [hunk 1]
```diff
@@ -50,11 +50,11 @@ public class SubscriptionsImportFragment extends BaseFragment {
     @State
     int currentServiceId = Constants.NO_SERVICE_ID;
 
-    private List<SubscriptionExtractor.ContentSource> supportedSources;
-    private String relatedUrl;
+    private List<SubscriptionExtractor.ContentSource> supportedSources = Collections.emptyList();
+    private String relatedUrl = null;
```
Comment (acrodemocide) [inline app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionsImportFragment.java:54]: I know this was originally set based on the function that was removed, but, if I recall correctly, a declared String in Java is already null, so this assignment is not needed.

Comment (Profpatsch) [reply app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionsImportFragment.java:54]: Probably, but this just inlines the existing code, which I don’t want to change here.

**Review by Stypox — APPROVED** [review]

File: app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionsImportFragment.java (hunk 1 above)
Comment (Stypox) [reply app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionsImportFragment.java:54]: Yeah there's no problem in making it explicit that it's initialized to null

**Comment by Stypox** [comment]
Sorry, I just realized that according to Android fragment lifecycle, the `onCreate` function might be called multiple times, so this PR actually does change semantics. I would not risk making this change right before the rewrite, as we want to ship as stable of a release as possible. However, it's true that the `setupServiceVariables` method could be inlined, but it's not too important, and will probably be rewritten sooner or later anyway.

**Pull request closed without merging**